ui_framework/src/components/page/_page_body.scss

```diff
@@ -0,0 +1,13 @@
+.kuiPageBody {
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  min-height: 400px; // Temporary till we have a better doc system.
```

Comment:
This is a non-destructive hack I have I needed so that I can show off the centering in the documentation layer. Once the doc tools allow me to view these on their own, a lot of heights will likely turn to 100%.

When that happens, if we decide to fix the header, we'll lightly adjust `.kuiPage` to compensate.